server/index.js

```diff
@@ -12,17 +12,83 @@ app.use(express.static(__dirname + '/../client/dist'));
 app.use(express.json());
 app.use(cors());
 
-
-app.get('/images', (req, res) => {
- console.log("request receiver") 
-	db.find()
+//get all images
+app.get('/images', (req, res) => { 
+	getImages()
   .then(images => {
-    res.json({
-  images
+    res.status(200).json({images})
+  })
+  .catch(err => {
+    res.status(200).send('unable to get all images');
+  })
```

Comment:
no error will come back if the database module doenst send back an error (see database file)

Reply:
same for below